### PR TITLE
Add a landing page for WTD Philly.

### PR DESCRIPTION
**Pull request by FelicianoTech**
Similar to: https://github.com/writethedocs/www/pull/98

### CODE REVIEW

File: docs/meetups/philly.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Write the Docs Philly
+===========================
+
+## About Us
+
+<img src="/_static/img/logos/writethedocsphl-logo.png" style="margin:0 0 20px 20px;width:200px;max-width:100%;float:right;" alt="Write the Docs Philly Logo" />
```
Comment (ericholscher) [inline docs/meetups/philly.md:6]: I love this logo -- are you using something in particular to generate them?

**Review by ericholscher — COMMENTED** [review]
Looks good to me. I know @plaindocs has thoughts on these pages, so he should get it out as well.

File: docs/meetups/philly.md (hunk 1 above)
Comment (plaindocs) [reply docs/meetups/philly.md:6]: :+1:

Comment (FelicianoTech) [reply docs/meetups/philly.md:6]: @ericholscher I made the NYC one in Gimp and then edited it for the Philly version. Here's the Gimp file: https://drive.google.com/open?id=0B0Mvprr0i5d_Y0Q3Qzk2YTZKR1U

**Comment by plaindocs** [comment]
Sorry for the delay.

**Pull request merged**